### PR TITLE
site: site style update

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: ChangeLog style update |
| 🇨🇳 Chinese | site: ChangeLog style update |

### CODE REVIEW

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const useStyle = createStyles(({ token }) => ({
     align-items: center;
     column-gap: ${token.paddingXXS}px;
     border-radius: ${token.borderRadiusSM}px;
-    padding-inline: ${token.paddingXXS}px;
+    padding-inline: ${token.paddingXXS}px !important;
```
Comment (li-jia-nan) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:49]: 更新前：
image

更新后：
image

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 2]
```diff
@@ -225,36 +231,40 @@ const ComponentChangelog: React.FC<ComponentChangelogProps> = (props) => {
...
+                          </li>
+                        ))}
+                      </ul>
+                    }
+                  >
+                    <BugOutlined className={styles.bug} />
+                  </Popover>
+                )}
+              </Typography.Title>
+              <Tag className={styles.versionTag} bordered={false} color="blue">
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:264]: 更新前：
image

更新后：
image

Comment (afc163) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:264]: image

list 的垂直间距有点小了，可以参考 changelog 页面调整一下。

Comment (li-jia-nan) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:267]: 看了一下，changelog 页面的 line-height 是 2，这里继承了 Typography 组件，所以变小了，我改成 2 了

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 3]
```diff
@@ -35,6 +35,11 @@ function matchDeprecated(v: string): MatchDeprecatedResult {
 }
 
 const useStyle = createStyles(({ token, css }) => ({
+  listWrap: css`
+    > li {
+      line-height: 2;
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:40]: 保持行高和 changelog 页面一致

**Review by afc163 — APPROVED** [review]

**Pull request merged**